server/src/main/java/io/druid/segment/realtime/appenderator/StreamAppenderatorDriver.java

```diff
@@ -378,10 +375,11 @@ public void onFailure(Throwable e)
       final Collection<String> sequenceNames
   )
   {
-    return Futures.transform(
-        publish(publisher, committer, sequenceNames),
-        this::registerHandoff
-    );
+    return ListenableFutures
+        .transformAsync(
```

Comment:
this line break doesn't seem necessary. It is a small thing but I still feel compelled to mention it.

Reply:
fixed